tests/zfs-tests/tests/functional/rsend/rsend_008_pos.ksh

```diff
@@ -38,7 +38,7 @@
 # STRATEGY:
 #	1. Separately promote pool clone, filesystem clone and volume clone.
 #	2. Recursively backup all the POOL and restore in POOL2
-#	3. Verify all the datesets and property be properly received.
+#	3. Verify all the datasets and property be properly received.
```

Comment:
```suggestion
#	3. Verify all the datasets and properties are properly received.
```

Reply:
Done!

Reply:
It still says "be" not "are" (but please make it "were" instead).

Reply:
Done!